CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [8.2.1] 2020-03-20
+
+### Added
+
+- First release.
+
+[Unreleased]: https://github.com/giantswarm/operatorkit/compare/v8.2.1...HEAD
```

Comment:
Wrong link here maybe

Reply:
Not maybe buddy. Thanks for catching that.